Apromore-Core-Components/Apromore-Portal/src/main/java/org/apromore/portal/common/Constants.java

```diff
@@ -94,9 +94,37 @@
     public static final String AUTH_SERVICE = "authorizationService";
     public static final String WORKSPACE_SERVICE = "workspaceService";
 
-    public static final String USER_ADMIN_PLUGIN = "USER_ADMIN_PLUGIN";
-    public static final String ACCESS_CONTROL_PLUGIN = "ACCESS_CONTROL_PLUGIN";
-    public static final String ETL_PLUGIN = "ETL_PLUGIN";
-    public static final String JOB_SCHEDULER_PLUGIN = "JOB_SCHEDULER_PLUGIN";
-
+    public static final String PLUGIN_USER_ADMIN = "USER_ADMIN";
+    public static final String PLUGIN_ACCESS_CONTROL = "ACCESS_CONTROL";
+    public static final String PLUGIN_ETL = "ETL";
+    public static final String PLUGIN_JOB_SCHEDULER = "JOB_SCHEDULER";
+    public static final String PLUGIN_CALENDAR = "CALENDAR";
```

Comment:
This Constants.java is becoming too big with different types of constants all live next to each other here. It is becoming hard to read, differentiate or modify safely different types of constants while they look quite similar. Suggest creating a new class called MenuLabelConstants to put many menu label constants here.